ogzaf/macro_params.py

```diff
@@ -11,6 +11,8 @@
 import numpy as np
 import datetime
 import statsmodels.api as sm
+from ogzaf import utils
```

Comment:
I needed to change this line to so that the `get_legacy_session` below was accessible without the `utils.` prefix:

```
from ogzaf.utils import get_legacy_session
```